educe/rst_dt/corpus.py

```diff
@@ -36,9 +35,20 @@ class Reader(educe.corpus.Reader):
     def __init__(self, corpusdir):
         educe.corpus.Reader.__init__(self, corpusdir)
 
-    def files(self):
+    def files(self, exclude_file_docs=False):
+        """
+        Parameters
+        ----------
+        exclude_file_docs : boolean, optional (default=False)
+            If True, fileX documents are ignored. The figures reported by
+            (Li et al., 2014) on the RST-DT corpus indicate they exclude
+            fileN files, whereas Joty seems to include them.
+            fileN documents are more damaged than wsj_XX documents, e.g.
+            text mismatches with the corresponding document in the PTB.
+        """
```

Comment:
Nice API comment! 

I'm using whatever sphinx-autodoc supports, for what it's worth.  I think the syntax looks like

```python
:param exclude_file_docs: explanation
                                           and more and more
:type exclude_file_docs: boolean
```

(not requesting you change this, just noting for future reference)